data/json/items/melee/swords_and_blades.json

```diff
@@ -3378,5 +3753,203 @@
+    "weight": "1400 g",
+    "volume": "1250 ml",
+    "longest_side": "60 cm",
+    "price": 97000,
+    "price_postapoc": 4000,
+    "to_hit": { "grip": "solid", "length": "short", "surface": "line", "balance": "neutral" },
+    "material": [ "lc_steel" ],
+    "repairs_with": [ "lc_steel" ],
+    "techniques": [ "WBLOCK_2", "RAPID" ],
+    "qualities": [ [ "CUT", 2 ], [ "BUTCHER", 7 ] ],
+    "flags": [ "SHEATH_SWORD", "CONDUCTIVE" ],
+    "weapon_category": [ "MEDIEVAL_SWORDS", "MEDIUM_SWORDS", "FENCING_WEAPONRY" ],
+    "melee_damage": { "bash": 5, "cut": 30 }
+  },
+  {
+    "id": "mc_talwar",
+    "type": "TOOL",
+    "copy-from": "lc_talwar",
+    "name": { "str": "medium steel talwar" },
+    "description": "A curved sword used by the Raj of india in the 16th century, a talwar is usually impeccibly decorated and able to take off a limb with a single swipe. \n\nThis one has a proper grip and is made of medium carbon steel, but doesn't have the distinctive circular pommel yet..",
+    "weight": "1450 g",
+    "volume": "1250 ml",
+    "price": 97000,
+    "price_postapoc": 4000,
+    "to_hit": { "grip": "solid", "length": "short", "surface": "line", "balance": "neutral" },
+    "material": [ "mc_steel" ],
+    "repairs_with": [ "mc_steel" ],
+    "techniques": [ "WBLOCK_2", "RAPID" ],
+    "qualities": [ [ "CUT", 2 ], [ "BUTCHER", 7 ] ],
+    "flags": [ "SHEATH_SWORD", "CONDUCTIVE" ],
+    "weapon_category": [ "MEDIEVAL_SWORDS", "MEDIUM_SWORDS", "FENCING_WEAPONRY" ],
+    "melee_damage": { "bash": 5, "cut": 34 }
+  },
+  {
+    "id": "hc_talwar",
+    "type": "TOOL",
+    "copy-from": "lc_talwar",
+    "name": { "str": "high steel talwar" },
+    "description": "A curved sword used by the Raj of india in the 16th century, a talwar is usually impeccibly decorated and able to take off a limb with a single swipe. \n\nThis one has a proper grip and is made of high carbon steel, you've even gotten the circular pommel right!",
+    "weight": "1450 g",
+    "volume": "1250 ml",
+    "price": 97000,
+    "price_postapoc": 4000,
+    "to_hit": { "grip": "weapon", "length": "short", "surface": "line", "balance": "neutral" },
+    "material": [ "hc_steel" ],
+    "repairs_with": [ "hc_steel" ],
+    "techniques": [ "WBLOCK_2", "RAPID", "BRUTAL" ],
+    "qualities": [ [ "CUT", 2 ], [ "BUTCHER", 7 ] ],
+    "flags": [ "SHEATH_SWORD", "CONDUCTIVE" ],
+    "weapon_category": [ "MEDIEVAL_SWORDS", "MEDIUM_SWORDS", "FENCING_WEAPONRY" ],
+    "melee_damage": { "bash": 5, "cut": 36 }
+  },
+  {
+    "id": "ch_talwar",
+    "type": "TOOL",
+    "copy-from": "lc_talwar",
+    "name": { "str": "case hardened steel talwar" },
+    "description": "A curved sword used by the Raj of india in the 16th century, a talwar is usually impeccibly decorated and able to take off a limb with a single swipe. \n\nThis one has a proper grip and is made of case hardened steel and is otherwise a fine weapon",
```

Comment:
```suggestion
    "description": "A curved sword used by the Raj of India in the 16th century, a talwar is usually impeccably decorated and able to take off a limb with a single swipe. \n\nThis one has a proper grip and is made of case hardened steel and is otherwise a fine weapon.",
```